tests/integration/test_design.py

```diff
@@ -2681,60 +2677,28 @@ def check_list_equality(lines, expected_lines):
     assert check_list_equality(lines, ref) is True
 
 
```

Comment:
Ah I just realized - something went wrong probably when updating your branch. I think you got rid of @jonahrb's test for surface body creation. Please restore it whenever you can.